pylutron/__init__.py

```diff
@@ -508,15 +508,16 @@ def send(self, op, cmd, integration_id, *args):
         (cmd, str(integration_id)) + tuple((str(x) for x in args if x is not None)))
     self._conn.send(op + out_cmd)
 
-  def load_xml_db(self, cache_path=None):
-    """Load the Lutron database from the server.
+  def load_xml_db(self, cache_path=None, refresh_data=True):
```

Comment:
Can you update the docstring with an args section:

Args:
  cache_path: Set to write the loaded XML to this file path.
  refresh_data: If set to false, will use the cache_path to load data if set.